splinepy/io/gismo.py

```
@@ -11,6 +11,297 @@
 
 
+class AdditionalBlocks:
+    """Class to manage additional XML blocks for gismo export, including boundary
+    conditions, function definitions, and assembly options. Each block is stored as a
+    Python dictionary, which is then later handled by the export function.
+    """
+
+    def __init__(self):
+        self._blocks = []
+
+    def add_boundary_conditions(
```

Pylint (3.3.7)

[refactor] Too many arguments (9/5)

(R0913)